std/container/array.d

```diff
@@ -539,11 +525,13 @@ Complexity: $(BIGOH 1)
     }
 
 /**
-Ensures sufficient capacity to accommodate $(D e) elements.
+Ensures sufficient capacity to accommodate $(D e) _elements.
+If $(D e < capacity), this method will not decrease the capacity.
 
 Postcondition: $(D capacity >= e)
 
-Complexity: $(BIGOH 1)
+Complexity: $(BIGOH e) if $(D e > capacity),
```

Comment:
If `realloc` can't extend old memory, it has to copy all the contents into the newly allocated space - I thought I should regard it as `O(capacity)`, or at least `O(length)`. Still, what if `realloc` extends it, does it counts as `O(1)` or `O(newCapacity - oldCapacity)`? Is there any official guideline in this regard?